app/test_main.py

```diff
@@ -1,3 +1,18 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (14, 14, [0, 0]),
+        (16, 9, [1, 0]),
+        (8, 17, [0, 1]),
+        (15, 15, [1, 1]),
+        (24, 24, [2, 2]),
+        (30, 31, [3, 3]),
+        (28, 28, [3, 2]),
```

Comment:
Test against zero, large, and negative numbers as mentioned in the checklist. Besides that, you could create a separate function to test against raised errors in case str, tuple, list or other non-supported data types are passed